File: qe-selenium-python/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Firefox()
driver.maximize_window()
driver.implicitly_wait(20)
wait = WebDriverWait(driver, 10)

driver.get("https://training-support.net/webelements/alerts")
print("Page Title: " + driver.title)

driver.find_element(By.XPATH, "//button[contains(text(),'Prompt')]").click()

wait.until(EC.alert_is_present())
alert = driver.switch_to.alert
print("Alert Text: " + alert.text)

alert.send_keys("Awesome!")
alert.accept()

driver.quit()

File: qe-java-selenium/Activity20.java
package demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.Alert;
import java.time.Duration;

public class Activity20 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        driver.manage().window().maximize();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(20));
        driver.get("https://training-support.net/webelements/alerts");

        System.out.println("Page Title: " + driver.getTitle());
        driver.findElement(By.xpath("//button[contains(text(),'Prompt')]")).click();
        Alert alert = driver.switchTo().alert();
        System.out.println("Alert Text: " + alert.getText());
        alert.sendKeys("Awesome!");
        alert.accept();
        driver.quit();
    }
}